### PR TITLE
feat(Drawer): added event param to onResize

### LINKED ISSUE #292: Drawer - added event param to onResize callback
Follow up to breaking change PR https://github.com/patternfly/patternfly-react/pull/8736

warn that an event param has been added to the onResize callback

_Required actions:_
1. Build codemod
2. Build test
3. Update readme with description & example

### PR DESCRIPTION
**Pull request by thatblindgeye**
Closes #292

### CODE REVIEW

**Comment by gitdallas** [comment]
should use the new helper function as well, right?

**Comment by thatblindgeye** [comment]
Yeah I'd need to rebase once my DataListCheck goes in as that should resolve some failing tests locally. I bumped this to the next milestone, though since I'll be out the rest of the week.

File: packages/pf-codemods/README.md [hunk 1]
```diff
@@ -412,6 +412,12 @@ Out:
 <Divider orientation={{ default: "vertical" }} />
 ```
 
+### drawer-warn-updated-callback [(#8736)](https://github.com/patternfly/patternfly-react/pull/8736)
+
+We've updated the `onResize` prop for Drawer to include the `event` as its first parameter. Handlers may require an update.
+
+This rule will raise a warning, but will not make any changes.
```
Comment (gitdallas) [inline packages/pf-codemods/README.md:419]: 😏

**Review by gitdallas — CHANGES_REQUESTED** [review]
readme could be updated after the fixers

File: packages/pf-codemods/README.md (hunk 1 above)
Comment (thatblindgeye) [reply packages/pf-codemods/README.md]: Good catch. Also updated the file names to be more specific (Drawer > DrawerPanelContent)

**Review by gitdallas — APPROVED** [review]

**Review by wise-king-sullyman — APPROVED** [review]

**Pull request merged**